.github/workflows/gh-pages-mdbook.yml

```diff
@@ -20,8 +23,8 @@ jobs:
       - name: Install Dependencies
         shell: bash
         run: sudo apt-get install -y libwayland-dev libxkbcommon-dev # Required for winit
-      - name: Install mdbook-alerts
-        run: cargo install mdbook-alerts
+      - name: Install mdbook-alerts, mdbook-mermaid
```

Comment:
you might want to use a cache-optimized https://github.com/marketplace/actions/cargo-install

Reply:
Our cache is pretty much non-functional, because it is way too small.

Also I am bit more held back including separate GitHub Actions after the last security incident.